Kernel/Syscalls/execve.cpp

```diff
@@ -626,6 +626,7 @@ KResult Process::do_exec(NonnullRefPtr<FileDescription> main_program_description
         ptrace_locker.unlock();
         Thread::current()->send_urgent_signal_to_self(SIGSTOP);
     }
+    ptrace_locker.unlock(); // unlock before disabling interrupts as well
```

Comment:
This is duplicated in the above `if` branch now. Seems like an opportunity to consolidate.